helpers/sql-obfuscation/lib/opentelemetry/helpers/sql_obfuscation.rb

```diff
@@ -42,29 +42,22 @@ class SqlObfuscation
         boolean_literals: /\b(?:true|false|null)\b/i,
         hexadecimal_literals: /0x[0-9a-fA-F]+/,
         comments: /(?:#|--).*?(?=\r|\n|$)/i,
-        multi_line_comments: %r{\/\*(?:[^\/]|\/[^*])*?(?:\*\/|\/\*.*)}
+        multi_line_comments: %r{\/\*(?:[^\/]|\/[^*])*?(?:\*\/|\/\*.*)}m,
```

Comment:
Why have we switched to a multiline match here? Actually, I find the last part of the original regex confusing - it basically matches `/*` followed by anything, eagerly, so in the context of the rest of the regex, we'd have `/* /*` and the regex would eat the rest of the input.

I feel like this regex is cleaner, simpler and more correct: `%r{(?:\/\*.*?\*\/)}m`. Is there a corner case this misses?

Reply:
Since the regex map referenced "multi_line_comments" in the name, I thought a multiline match would be an appropriate addition. I don't mind removing it though if you disagree. 

I couldn't find a corner case that your simpler regex missed. I'll update the code to use that instead.

Reply:
Updated in 7b900745